### PR TITLE
Ability to disable publishing to the Pulumi CDN

### PR DESCRIPTION
**Pull request by ringods**
Ability to disable publishing to the Pulumi CDN (`get.pulumi.com`) for non-Pulumi managed providers.

Ran `make test-providers` which didn't render any changes, but it worked for my `pulumiverse/pulumi-acme` provider.

### CODE REVIEW

**Comment by guineveresaenger** [comment]
I apologize for the question, but what is Pulumi CDN?

This looks reasonable to me otherwise - I would like to see a third-party provider test in the repo maybe, so we can see the applied changes alongside the PR.

**Comment by ringods** [comment]
@guineveresaenger I updated the initial comment. Pulumi CDN is `get.pulumi.com` which is not accessible for 3rd party packages.

File: provider-ci/internal/pkg/templates/bridged-provider.config.yaml [hunk 1]
```diff
@@ -175,6 +175,8 @@ publish:
   # passed to the sdk input of pulumi-package-publisher
   # This is overridden in pulumi-local to disable python
   sdk: all
+  # Publish the plugin binaries to the Pulumi CDN
```
Comment (danielrbradley) [inline provider-ci/internal/pkg/templates/bridged-provider.config.yaml:178]:
```suggestion
  # Publish the plugin binaries to the Pulumi CDN (get.pulumi.com) - requires AWS credentials for S3 upload
```

**Review by danielrbradley — APPROVED** [review]
Happy this is a nice safe change. I think this is very specifically just an s3 upload ... it's just this s3 bucket happens to be behind a cdn. I think with the expanded comment it's fine though.

**Comment by ringods** [comment]
@guineveresaenger `acme` added as test provider.

**Pull request merged**